webapp/jsp/admin/plugins/blog/ManageAdminDashboard.jsp

```diff
@@ -0,0 +1,25 @@
+<%@ page import="static fr.paris.lutece.plugins.blog.web.utils.BlogConstant.PARAMETER_PLUGIN_NAME" %>
+<%@ page import="static fr.paris.lutece.plugins.blog.web.utils.BlogConstant.PARAMETER_BLOG" %>
+<%@ page import="static fr.paris.lutece.plugins.blog.web.utils.BlogConstant.PARAMETER_MANAGE_MAX_PUBLICATION_DATE" %>
+<%@ page import="static fr.paris.lutece.plugins.blog.web.utils.BlogConstant.PARAMETER_ACTION" %>
+<%@ page import="static fr.paris.lutece.plugins.blog.web.utils.BlogConstant.PARAMETER_MANDATORY_TAG_NUMBER" %>
+<%@ page import="static fr.paris.lutece.plugins.blog.web.utils.BlogConstant.PARAMETER_ACTION" %>
+
+<jsp:useBean id="manageAdminDashboard" scope="session" class="fr.paris.lutece.plugins.blog.web.adminDashboard.BlogAdminDashboardJspBean" />
```

Comment:
Why is the redirection management handled in the JSP, rather than in the BlogAdminDashboardJspBean class?